org_chart.js

```diff
@@ -1,73 +1,132 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
+"use strict";
+
+function Employee(name, email, position, gender, project){
+  this.name = name;
+  this.email = email;
+  this.position = position;
+  this.gender = gender;
+  if (!project) {
+  this.project = "";
+  } else {
+  this.project = project;
+  }
 }
 
-let new_hires = [
-  {
-    name: "Alan Turing",
-    email: "alan.turing@musky.com",
-    position: "Computer Scientist",
-    project: "Cryptography"
-  },
-  {
-    name: "Grace Hopper",
-    email: "grace.hopper@musky.com",
-    position: "Computer Scientist",
-    project: "Compilers"
-  },
-  {
-    name: "Donald Knuth",
-    email: "donald.knuth@musky.com",
-    position: "Mathematician",
-    project: "Algorithm Analysis"
+Employee.prototype.addSkill = function(skills) {
+  if (!this.skills) {
+    this.skills = [];
+  } else {
+  this.skills = skills;
   }
-]
+}
 
-// assign each a manager
-let [alan, grace, donald] = new_hires
+Employee.prototype.setManager = function(manager) {//not working?
+  this.manager = manager;
+}
 
-alan.manager = elon
-grace.manager = elon
-donald.manager = elon
+Employee.prototype.print = function(){ //working
+  console.log(`${this.name} is a ${this.position} working on ${this.project}.`)
+  if (this.gender === "female") {
+      console.log(`She can be reached at ${this.email}`)
+  } else {
+  console.log(`He can be reached at ${this.email}`)
+  }
```

Comment:
All you needed was to determine "She" or "He" instead of the whole string.
The ternary operator would've been perfect here.